helm-chart/dash0-operator/templates/operator/custom-resource-definition-monitoring.yaml

```diff
@@ -397,6 +405,34 @@ spec:
                 description: Shows results of synchronizing Prometheus rule resources
                   in this namespace via the Dash0 API.
                 type: object
+              viewSynchronizationResults:
```

Comment:
suggestion: This change in the monitoring resource should also follow the Go-code first approach outlined above.